src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1189,6 +1189,11 @@ private async Task HandleBreakModeEvent(ResultEventArgs results, BreakRequest br
                     bplist.AddRange(bkpt);
                     _callback.OnBreakpoint(thread, bplist.AsReadOnly());
                 }
+                else if (ExceptionManager.TryGetExceptionBreakpoint(bkptno, out EXCEPTION_INFO exceptionInfo) == Constants.S_OK) // exception breakpoint hit
```

Comment:
>== Constants.S_OK [](http://example.com/codeflow?start=110&length=17)

nit: I would suggest returning a `bool` instead of an HRESULT. Both for methods that our "bool sounding" like "Try..." and also just for internal methods in general when in C# code.